README.md

```diff
@@ -43,7 +43,13 @@ or run the individual commands from the Procfile in separate terminals.
 Note: if you installed the Ruby version of foreman, you'll need to use `foreman start`.
 
 ## Testing
-WIP
+The App Scaffold is currently setup for testing with [Jasmine](http://jasmine.github.io/) (testing framework) and [Karma](https://karma-runner.github.io) (test runner). To run specs, run
+
+```
+karma start
```

Comment:
Yes, added to setup instructions on https://github.com/zendesk/app_scaffold/pull/17/commits/77ba887e660db6e6ea5263a24811d3c54681ad9e